doc/sphinx/src/models.rst

```diff
@@ -1161,3 +1163,61 @@ See :ref:`EOSPAC Vector Functions <eospac_vector>` for more details.
 .. _Sesame: https://www.lanl.gov/org/ddste/aldsc/theoretical/physics-chemistry-materials/sesame-database.php
 
 .. _EOSPAC: https://laws.lanl.gov/projects/data/eos/eospacReleases.php
+
+Stiffened Gas
+`````````````
+
+The following implementation was guided by the reference `StiffGas`_. The stiffened gas model in ``singularity-eos`` takes
+the form
+
+.. math::
+
+    P = \Gamma \rho (e-q) - \gamma P_{\infty}
+
+.. math::
+
+    e = \frac{ P + \gamma P_{\infty} }{ P + P_{\infty} } C_V T + q,
+
+where quantities are largely the same as ideal gas with the exception of :math:`P_{\infty}` and :math:`q`, which can be thought of as an offset pressure and internal energy, respectively.
+
+The entropy for a stiffened gas can be expressed as
+
+.. math::
+
+    S = C_V \ln\left(\frac{T}{T_0}\right) + (\gamma-1) C_V \ln\left(\frac{\rho_0}
+     {\rho}\right) + q',
+
+we have assumed that the entropy and offset :math:`q'` are zero at the reference state given
```

Comment:
I think you mean
```
where :math:`S(\rho_0, T_0) = q'`.
```
This is also less verbose and I think gets the point across.

Reply:
Corrected